<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>barankrky</author>
 <class>UserManager</class>
 <widget class="QMainWindow" name="UserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="app_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>User Manager</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>101</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCommandLinkButton" name="loginbtn">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="registerbtn">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="exitbtn">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="version_lbl">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Version :</string>
    </property>
   </widget>
   <widget class="QLabel" name="version_number">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
